src/topology/separation.lean

```diff
@@ -554,3 +554,66 @@ begin
 end
 
 end normality
+
+/-- In a compact t2 space, the connected component of a point equals the intersection of all
+its clopen neighbourhoods. -/
+lemma connected_component_eq_Inter_clopen [t2_space α] [compact_space α] {x : α} :
+  connected_component x = ⋂ Z : {Z : set α // is_clopen Z ∧ x ∈ Z}, Z :=
+begin
+  apply eq_of_subset_of_subset,
+  { exact connected_component_subset_Inter_clopen},
```

Comment:
To minimize a little bit the nesting level, you can clear directly the simple goals. For instance, here you can write
```suggestion
  apply eq_of_subset_of_subset connected_component_subset_Inter_clopen,
```
and then there is one single goal remaining, which means that the next accolade can be removed and everything can be unindented by two spaces.